baselines/dom.generated.d.ts

```diff
@@ -921,10 +923,11 @@ interface PromiseRejectionEventInit extends EventInit {
     reason?: any;
 }
 
-interface PropertyIndexedKeyframes extends Record<keyof CSSStyleDeclaration, string | string[]> {
-    composite?: CompositeOperation | (CompositeOperation | null)[];
+interface PropertyIndexedKeyframes {
+    composite?: CompositeOperationOrAuto | CompositeOperationOrAuto[];
     easing?: string | string[];
     offset?: number | (number | null)[];
+    [property: string]: string | string[] | number | null | (number | null)[] | undefined;
```

Comment:
Could you explain the `(number | null)[]` thing? Do we also need `(string | null)[]`?

Reply:
> the type of indexed properties can't conflict with the type of other properties specified on the interface

Ah, that's right. Thanks!